phpBB/includes/acp/acp_storage.php

```diff
@@ -78,14 +97,17 @@ public function main($id, $mode)
 		 */
 		$phpbb_dispatcher->dispatch('core.acp_storage_load');
 
-		$this->overview($id, $mode);
+		@ini_set('memory_limit', '128M');
```

Comment:
Im' not sure you should do that here

Reply:
Is a copy/paste https://github.com/phpbb/phpbb/blob/master/phpBB/includes/acp/acp_search.php#L37

Reply:
ok